arboretum/permissions/README.md

```diff
@@ -94,6 +94,55 @@ how to include the fetchers and checks from this library in your downstream proj
 
 ## Checks
 
+### Organization Integrity (Repository Permissions)
+
+* Class: [GithubOrgPermissionsCheck][org-permissions-check]
+* Purpose: Control the access to Github repositories containing source code.
+* Behavior: Collaborators, fork and teams are checked for every repository.
+Each direct collaborator found in a repo will be considered a failure. Each fork found in a repo will be considered a warning. As additional information the permissions check also lists the organization teams as successes.
+* Evidence depended upon:
+    * `direct` collaborators found in organization repositories.
+      * `raw/permissions/gh_direct_collaborators_<org_url_hash>.json`
+      * Gathered by the permissions GithubOrgPermissionFetcher
+    * `outside` collaborators not found within the organizations.
+      * `raw/permissions/gh_outside_collaborators_<org_url_hash>.json`
+      * Gathered by the permissions GithubOrgPermissionFetcher
+    * forks found in organization repositories.
+      * `raw/permissions/gh_forks_<org_url_hash>.json`
+      * Gathered by the permissions GithubOrgPermissionFetcher
+    * teams found in organization repositories.
+      * `raw/permissions/gh_teams_<org_url_hash>.json`
+      * Gathered by the permissions GithubOrgPermissionFetcher
```

Comment:
- `GithubOrgPermissionFetcher` should be a link
- Why are "all" collaborators excluded from this check?

Reply:
Done